Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -60,15 +61,16 @@ private enum AssetCatalog {
   }
 
   enum Item {
+    static let colorSet = "colorset"
     static let imageSet = "imageset"
 
     /**
      * This is a list of supported asset catalog item types, for now we just
-     * support `image set`s. If you want to add support for new types, just add
-     * it to this whitelist, and add the necessary code to the
-     * `process(items:withPrefix:)` method.
+     * support `image set`s and `color set`s. If you want to add support for
+     * new types, just add it to this whitelist, and add the necessary code to
+     * the `process(items:withPrefix:)` method.
      */
-    static let supported = [imageSet]
+    static let supported = [colorSet, imageSet]
```

Comment:
So glad we handled that whitelist, we were ready already 👍 

I'm wondering though if we shouldn't have used `cases` here, i.e.

```swift
enum Item: String
  case colorSet
  case imageSet

  static let supported: [Item] = [.colorSet, .imageSet]
```

That way would would have ensured at compile time that everywhere we have a `switch` to handle each case we never forget to handle every supported cases and we have an exhaustive `switch` never forgetting places to add new code with new types being supported.

Of course that would mean handling the case of standard folders without extension (the last `case` on line 110) outside the `switch`, like in a separated `if` block or whatnot, but at least that would add safety provided by the compiler, so…